client/client.go

```diff
@@ -145,6 +146,10 @@ func newHttpsClient(conf Config) (*http.Client, error) {
 
 	trustedcerts, err := loadServerTrust(conf)
 	if err != nil {
+		log.Error("Can not open certificate file.")
+		log.Error("Your /etc/mender/server.crt can not be opened, if you are using " +
```

Comment:
You don't know if that's the case. See also the first comment in `loadServerTrust`.

I would remove this whole logging section and just keep returning the error, since you already get quite a lot of information from inside `loadServerTrust`, including more information about *why* there was an error. If you want to include the "make sure to include API gateway .... " message, then perhaps it can be moved there instead.

Reply:
Right. Fixing!